docs/wxm-devices/helium-bundle/helium-firmware-updade.mdx

```diff
@@ -14,7 +14,7 @@ if you experience connection issues while claiming a helium device,  make sure y
 :::
 
 <div class="explanation small-big">
-  <img src={useBaseUrl("img/wxm-devices/helium-bundle/update-station.svg")} />
+  <img src={useBaseUrl("img/wxm-devices/helium-bundle/update-station.jpg")} />
 <span>
```

Comment:
@epapangelis Please delete the old .svg file if it's not needed anymore in any other occasion.